examples/policies/l3/service/service-labels.yaml

```diff
@@ -11,4 +11,4 @@ spec:
     - k8sServiceSelector:
         selector:
           matchLabels:
-            external: "yes"
+            bar: "bar"
```

Comment:
nit: Maybe this should instead use something like `serviceName: myservice`. This would match with `examples/policies/l3/service/service.yaml`.

Reply:
Done.